### PR TITLE
[B] Fixes the Duration-Value in /effect command, Fixes BUKKIT-3967

### PR DESCRIPTION
**Pull request by martin3398**
The Issue:
When Using the /effect command. The Time is outputted in Ticks not in seconds.

PR Breakdown:
during the /effect command a variabnle (duration_sec) will be declared and initalized with duration / 20

JIRA-Ticket:
BUKKIT-3967 - https://bukkit.atlassian.net/browse/BUKKIT-3967

### CODE REVIEW

File: src/main/java/org/bukkit/command/defaults/EffectCommand.java [hunk 1]
```diff
@@ -62,6 +62,7 @@ public boolean execute(CommandSender sender, String commandLabel, String[] args)
         }
 
         int duration = 600;
+        int duration_sec;
```
Comment (riking) [inline src/main/java/org/bukkit/command/defaults/EffectCommand.java:65]: Why'd you add the variable way up here?

Why not just put it inline?

Comment (martin3398) [inline src/main/java/org/bukkit/command/defaults/EffectCommand.java:65]: the variable is just temopory for  later changes...

**Comment by EdGruberman** [comment]
Please follow the PR formatting guidelines.

What advantage does this pull have over #844?

**Comment by turt2live** [comment]
Closing in favour of #844

**Pull request closed without merging**